base/utils/data/standard_vars.csv

```diff
@@ -106,7 +106,7 @@ Tmin,air_temperature_max,K,Daily Maximum Temperature,Driver,real,lon,lat,time,NA
 Tmax,air_temperature_min,K,Daily Minimum Temperature,Driver,real,lon,lat,time,NA,Daily Minimum Temperature,
 Uwind,northward_wind,m s-1,Northward Component of Wind,Driver,real,lon,lat,time,NA,Northward Component of Wind,
 Vwind,eastward_wind,m s-1,Eastward Component of Wind,Driver,real,lon,lat,time,NA,Eastward Component of Wind,
-RH,relative_humidity,%,Relative Humidity,Driver,real,lon,lat,time,NA,Relative Humidity,
+RH,relative_humidity,1,Relative Humidity,Driver,real,lon,lat,time,NA,Relative Humidity,
```

Comment:
Checking to see if this is the cause of the failed test. https://github.com/PecanProject/pecan/actions/runs/17193877078/job/48773185202?pr=3604
```suggestion
RH,relative_humidity,%,Relative Humidity,Driver,real,lon,lat,time,NA,Relative Humidity,
```